wireguard-vpn/js/view/Tunnels.js

```diff
@@ -235,3 +244,20 @@ Ext.define('Ung.apps.wireguard-vpn.cmp.TunnelsGrid', {
         }]
     }]
 });
+
+function isUnique(value, field, component) {
+    var currentRecord = component.up('window').getViewModel().data.record.get(field);
+    var grid = Ext.ComponentQuery.query('app-wireguard-vpn-server-tunnels-grid')[0];
+    var store = grid.getStore();
+
+    var isNameUnique = store.findBy(function(record) {
+        return record.get(field) === value;
+    }) === -1;
+    
+    if (value === currentRecord || isNameUnique) {
+        return true;
+    } else {
+        var capitalizedField = field.charAt(0).toUpperCase() + field.slice(1);
+        return Ext.String.format('{0} already exists.'.t(), capitalizedField);
```

Comment:
This message on an Add window might not give more insights on where this field already exists.
How about this message ?
```
Tunnel with this ${field} already exists.